tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -54,29 +58,105 @@ struct SynthOrdered {
     }
 };
 
-template <class T>
-inline constexpr bool is_pair = false;
-template <class A, class B>
-inline constexpr bool is_pair<std::pair<A, B>> = true; // TRANSITION, std::pair spaceship not yet implemented
+struct OrderedChar {
+    OrderedChar() = default;
+    OrderedChar(const char c) : c(c){};
 
-template <class Container>
-void ordered_containers_test(const Container& smaller, const Container& smaller_equal, const Container& larger) {
+    OrderedChar& operator=(const char& other) {
+        this->c = other;

```

Comment:
We don't conventionally use extraneous `this->` for members; the anomaly makes me think I'm missing some funny dependent base shenanigans.

Reply:
Nope, it was an artifact of a previous implementation. It's no longer needed.